static/js/app/store.js

```diff
@@ -0,0 +1,25 @@
+import {createStore, applyMiddleware, compose} from 'redux';
+import {browserHistory} from 'react-router';
+import {routerMiddleware} from 'react-router-redux';
+import {createLogger} from 'redux-logger';
+import rootReducer from './reducer';
+
+
+const middlewares = [routerMiddleware(browserHistory)];
+
+if(process.env.NODE_ENV !== 'production') {
```

Comment:
create a separate file with constants to hold all environments

Reply:
@python4eg separate file with environments constants was [created](https://github.com/Social-projects-Rivne/Rv-026WebUI/blob/b99e987113669797db23af493b2faa4f793afbb7/config.js)!